gtk/src/dialogs/mod.rs

```diff
@@ -1,3 +1,5 @@
+extern create textwrap;
```

Comment:
I believe this is what is causing the build to fail. Change `create` to `crate` and it should work better.